content/post/wxs-reflection-002.md

```diff
@@ -0,0 +1,44 @@
++++
+title = 'On Time'
+date = 2024-05-30T12:00:00-04:00
+authors = ["Xavier Snelgrove"]
++++
+
+This week has been a less explicitly productive one for my research. However because of the auto-ethnographic quality to the work I'm doing, technically anything I do while inside the fellowship container is data on what it's like to be inside the fellowship container...
+
+It's hard to believe it's actually only been a week since our last meeting. Time has a funny way of bending and stretching when in these kinds of spaces. There's a currently fashionable ancient conception of time as having two faces: Chronos which is "clock time", the sort of linear sequential time in which one makes plans and calendar events, the sort of time that under our economic system one sells, the kind that we're always running out of. And then Kairos, which is the sort of atemporal circular multidimensional time of our direct experience, the time which expands when you're deep in the flow of conversation, the time which is contracted by the tyranny of a meeting agenda, the time which as you watch the sunset connects you to every other sunset you've watched.
+
+These two kinds of time are in a sort of tension with one another. Chronos has this quality of practical necessity: people with a poor handle on Chronos tend to be chronically[^1] late, have trouble delivering complex projects. Kairos of a humane and cosmic empathy. People with a poor handle on Kairos tend towards burnout, for themselves and their collaborators, tend to be rigid in their thinking and planning having trouble adapting to changing circumstances, and may not respond well to the information held in intuitions and the natural ebb and flow of their energy.
```

Comment:
I went down a rabbit hole to find words that derive from Kairos in Italian (the Chronos ones are very obvious), and in doing so I came across [a third greek deity related to time](https://en.wikipedia.org/wiki/Aion_(deity)) -  I believe Aion is associated with a definition of time that's unbound (such a good word!) and cyclical - this idea of a returning version of the past, and I am curious of how this new "god" would fit into this thinking (if at all)

Reply:
Ooh, I love this, had not come across Aion, but I do love cyclical time... 

There's this idea of thinking about "7 generations" as a way to connect to deeper time. I think it was in the book "sand talk" that I learned how that is grounded in the timespan of living memory, that is: if great grandparents are pretty much the limits of lifespan, then a great grandparent can remember their great grandparents, their grandparents, their parents, themselves, their children, their grandchildren, and their great grand children.

In the book as I recall he says his great grandmother used to call him "great grandfather" thereby explicitly connecting the two horizons given by the limits of mortality into one cycle...